src/stores/MessageDisplayRulesStore.ts

```diff
@@ -134,31 +135,52 @@ class MessageDisplayRulesStore {
 				session: '*',
 				viewType: MessageViewType.JSON,
 				removable: false,
-				fullyEditable: false,
+				editableSession: false,
+				editableType: true,
 				timestamp: moment.utc().valueOf(),
 			};
 			this.indexedDb.addDbStoreItem(IndexedDbStores.DISPLAY_RULES, rootDisplayRule);
 		} else {
 			rootDisplayRule = displayRules.splice(rootRuleIndex, 1)[0];
-		}
-
-		displayRules.sort((ruleA, ruleB) => {
-			let indexA = order.indexOf(ruleA.session);
-			let indexB = order.indexOf(ruleB.session);
-
-			indexA = indexA === -1 ? Number.MAX_SAFE_INTEGER : indexA;
-			indexB = indexB === -1 ? Number.MAX_SAFE_INTEGER : indexB;
-
-			if (indexA > indexB) {
-				return 1;
-			}
-
-			if (indexB > indexA) {
-				return -1;
+			// getting rid of deprecated rule property
+			// eslint-disable-next-line no-prototype-builtins
+			if (rootDisplayRule.hasOwnProperty('fullyEditable')) {
```

Comment:
this should have been done in indexeddb upgade, but lets leave this property for now